torch/csrc/jit/codegen/cuda/arith.cpp

```diff
@@ -186,6 +186,58 @@ Val* newValLike(Val* val, DataType dtype) {
   return newScalar(vtype, dtype);
 }
 
+Val* getMinimumValue(DataType v) {
```

Comment:
Not a requirement, but a brief comment for each function would be nice for readability/reviewability